hw/ip/rv_dm/data/rv_dm_testplan.hjson

```diff
@@ -463,6 +463,22 @@
       stage: V1
       tests: ["rv_dm_halt_resume_whereto"]
     }
+    {
+      name: progbuf_read_write_execute
+      desc: '''
+            Verify the functionality of program buffer by writing random data on program buffer.
+
+            - Program jtag_dmi_ral.dmcontrol.haltreq to 1'b1.
+            - Acknowledge haltreq by setting tl_mem_ral.halted to 0.
+            - Write random data on jtag_dmi_ral.pragram_buffer_* registers to access
+              program buffer.
```

Comment:
What does the "to access program buffer" bit mean? I'm not sure it's required.

Probably something like this would be simpler? "Fill the program buffer by writing random data to `jtag_dmi_ral.program_buffer_*` registers."